test/e2e/webdriver/README.md

```diff
@@ -0,0 +1,1396 @@
+# About Driver
+
+Driver is a custom class designed for handling web automation interactions, serving as a wrapper around the Selenium WebDriver library.
+
+## Driver key features:
+
+- Locator strategy that utilizes the buildLocator function, supporting inline locators as an alternative to the traditional use of Selenium's By abstraction.
+- The finding element calls the wrapElementWithAPI function which mirrors the Playwright API, facilitating tool migration.
+- A comprehensive suite of functions for element, interacting with them, and performing actions using keyboard and mouse.
+- Appropriate waiting strategies for elements to appear within a time period or condition is met.
+- Management of browser windows, tabs, alerts, and frames with appropriate navigation, switching, and closing capabilities.
+- Validation of the application with assertion statements to check expected values and conditions.
+- Error-handling mechanisms to capture and log browser console errors, simplifying the identification and troubleshooting of issues encountered during testing.
+- Capture of screenshots during automated testing in the event of failures, aiding in debugging and issue pinpointing.
+
+## Locators
+
+In web automation testing, locators are crucial commands that guide the framework to identify and select [HTML elements](https://www.w3schools.com/html/default.asp) on a webpage for interaction. They play a vital role in executing various actions such as clicking buttons, fill text, or retrieving data from web pages. Gaining a solid understanding of locators is a key step in initiating web testing automation, as they form the foundation for engaging with web elements.
+
+## buildLocator
+
+**`buildLocator`** function enhances element matching capabilities by introducing support for inline locators, offering an alternative to the traditional use of Selenium's By abstraction.
+
+[source](https://github.com/MetaMask/metamask-extension/blob/1f2bfb388695034db8859877ed21b4b045514f9f/test/e2e/webdriver/driver.js#L190)
+
+**Arguments**<br>
```

Comment:
```suggestion
**Arguments**
```
There's a fair amount of `<br>`s here - are they necessary in this markdown section?